.github/workflows/contrib_checks.yml

```diff
@@ -97,7 +97,7 @@ jobs:
   rs-lints:
     name: Rust lints (fmt, check, clippy, tests, doc)
     # TODO(andreas): setup-vulkan doesn't work on 24.4 right now due to missing .so
-    runs-on: ubuntu-22.04-large
+    runs-on: buildjet-16vcpu-ubuntu-2204
```

Comment:
Are we sure we can run contributor checks on buildjet?

Reply:
no idea, but why not?